wear/src/main/java/io/homeassistant/companion/android/util/CommonFunctions.kt

```diff
@@ -143,6 +142,14 @@ fun getIcon(entity: Entity<Map<String, Any>>?, domain: String, context: Context)
     }
 }
 
+fun getIconFromMdiString(icon: String, context: Context?): IIcon {

```

Comment:
Due to the latest changes `Context?` here can now also be `Context`, and lines 147-149 can be removed.

Reply:
Aahh, got it! I have much better understanding of null assertions in Kotlin now, after all of these changes.
Many thanks for the tyour time and help :)